src/Runner.Worker/ExecutionContext.cs

```diff
@@ -893,11 +893,19 @@ public static void Debug(this IExecutionContext context, string message)
             }
         }
 
-        public static PipelineTemplateEvaluator ToPipelineTemplateEvaluator(this IExecutionContext context)
+        public static IEnumerable<KeyValuePair<string, object>> ToExpressionState(this IExecutionContext context)
```

Comment:
Values to put into the TemplateContext.State dictionary which is provided to functions that are evaluated